<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>downloader</class>
 <widget class="QMainWindow" name="downloader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>downloader</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="urlBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>531</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>URL</string>
    </property>
    <widget class="QLineEdit" name="url">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>www.site.fr/</string>
     </property>
    </widget>
    <widget class="QLabel" name="suivi">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>puis chap/image :</string>
     </property>
    </widget>
    <widget class="QLabel" name="imageLabel">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>20</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>42/001.jpg</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="optionsDlBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>531</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Options de téléchargement</string>
    </property>
    <widget class="QGroupBox" name="zeroBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>161</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>0 non significatifs</string>
     </property>
     <widget class="QLineEdit" name="nombre">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLabel" name="zeroLabel1">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>001.jpg</string>
      </property>
     </widget>
     <widget class="QLabel" name="zeroLabel2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>010.jpg</string>
      </property>
     </widget>
     <widget class="QLabel" name="zeroLabel3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>100.jpg</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="minMaxBox">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>161</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Min et max</string>
     </property>
     <widget class="QLineEdit" name="minNumber">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="maxNumber">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>150</string>
      </property>
     </widget>
     <widget class="QLabel" name="maxLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Max :</string>
      </property>
     </widget>
     <widget class="QLabel" name="minLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Min :</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="extBox">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>161</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Extension</string>
     </property>
     <widget class="QLineEdit" name="extLineEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>jpg</string>
      </property>
     </widget>
     <widget class="QLabel" name="extLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Indiquer l'extension :</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="optionsSaveBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>531</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Options de sauvegarde</string>
    </property>
    <widget class="QLineEdit" name="dossier">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>dossier</string>
     </property>
    </widget>
    <widget class="QLabel" name="dossierLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Dossier de sauvegarde :</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="zeroSaveBox">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>161</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>0 non significatifs</string>
     </property>
     <widget class="QLineEdit" name="nombreSave">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLabel" name="zeroSaveLabel1">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>001.jpg</string>
      </property>
     </widget>
     <widget class="QLabel" name="zeroSaveLabel2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>010.jpg</string>
      </property>
     </widget>
     <widget class="QLabel" name="zeroSaveLabel3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>100.jpg</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="download">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>580</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Télécharger</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>580</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="actual">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>620</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>image en cours</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="chapOtionBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>531</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Options de chapitre</string>
    </property>
    <widget class="QGroupBox" name="chapZeroBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>161</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>0 non significatifs</string>
     </property>
     <widget class="QLineEdit" name="chapNombre">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="chapZeroLabel1">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="chapZeroLabel2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QLabel" name="chapZeroLabel3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>100</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="chapMinMaxBox">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>161</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Min et max</string>
     </property>
     <widget class="QLineEdit" name="chapMinNumber">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="chapMaxNumber">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="chapMaxLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Max :</string>
      </property>
     </widget>
     <widget class="QLabel" name="chapMinLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Min :</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>568</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
